example/frontend/Dockerfile

```diff
@@ -11,6 +11,7 @@ RUN apk add git &&\
 FROM nginx:alpine
 
 COPY --from=builder /build/build /usr/share/nginx/html
+RUN rm -rf /usr/share/nginx/html/sw.js
```

Comment:
Why is this necessary? Hasn't this already been removed in line 9?

Reply:
It has indeed and it's not necesary, thank you!